truffle.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
     development: {
       host: 'localhost',
       port: 8545,
-      network_id: '*', // eslint-disable-line camelcase
+      network_id: '*' // eslint-disable-line camelcase
```

Comment:
was this enforced by our eslint config or your editor? I think our eslint config enforces the trailing multiline comma